src/stores/tabs/common.ts

```diff
@@ -26,6 +27,7 @@ export const getDefaultTab = (): Tab => ({
   config: {
     theme: "dark",
     notifications: true,
+    media: false,
```

Comment:
actually I think it makes sense to have this be on by default since most users will want it to work without having to turn on a setting.

i've already merged the PR, so I'll just make that change on main before I create the release

Reply:
I agree, it makes sense the default to be enabled!